index.bs

```diff
@@ -202,26 +202,15 @@ The OrientationSensor Interface {#orientationsensor-interface}
 
 <pre class="idl">
   interface OrientationSensor : Sensor {
-    void populateQuaternion(Float32Array targetBuffer);
+    readonly attribute FrozenArray<<double>>? quaternion;
```

Comment:
Can you try this readonly attribute FrozenArray&lt;double>? quaternion;
Instead of putting <<>>